external-import/comlaude/src/main.py

```diff
@@ -285,16 +305,13 @@ def run(self):
             self.comlaude_search.get_next_page()
             self._iterate_events()
 
-        if self.helper.connect_run_and_terminate:
-            self.helper.log_info(
-                "Connector stop: ({})".format(
-                    _format_time(datetime.datetime.now(datetime.UTC))
-                )
+        self.helper.log_info(
```

Comment:
Just for information! The new way to log is in this format: 
Ex: self.helper.connector_logger.info("your message", {"foo": foo, "bar": bar})

Reply:
Hello, ok it's noted thanks for information